.infra/index.ts

```diff
@@ -63,8 +63,8 @@ if (isAdhocEnv) {
 const redis = new Redis(`${name}-redis`, {
   isAdhocEnv,
   name: `${name}-redis`,
-  tier: 'STANDARD_HA',
-  memorySizeGb: 10,
+  tier: 'BASIC',
+  memorySizeGb: 2,
```

Comment:
What happens if we go over?

Reply:
🤷‍♂️

It'll probably not let us write any more

Reply:
Maybe put a bit more breathing space, 3Gb?

Reply:
It starts evicting keys depending on our policy

Reply:
Looking at the code I think marketing CTA is the only one that could bump the usage drastically if we start using it due to a lot of potential redis keys.

Reply:
We can easily scale up if we see marketing CTA starts to eat more memory.